src/main/kotlin/snyk/common/SnykCachedResults.kt

```diff
@@ -197,12 +194,40 @@ class SnykCachedResults(
                                 )
                         }
                     }
+
+                    val errorResponse = Gson().fromJson(snykScan.errorMessage, ErrorResponse::class.java)
```

Comment:
nitpick: am not sure we need to deserialize here. how were we getting the error from the diagnostics response call before?

Reply:
Previously, it was displaying the whole object and thus more information. We should not lose that information:
- Folder Path
- Product
- Error Message

So yeah, maybe we don't need to transform, it and just pass `$snykScan` as before.

Reply:
Will revert this change.